e2e/__tests__/toMatchInlineSnapshotWithPretttier3.test.ts

```diff
@@ -66,10 +64,10 @@ test('supports passing `null` as `prettierPath`', () => {
   expect(exitCode).toBe(0);
 });
 
-test('supports passing `prettier-2` as `prettierPath`', () => {
+test('supports passing `prettier` as `prettierPath`', () => {
   writeFiles(DIR, {
     'jest.config.js': `
-        module.exports = {prettierPath: require.resolve('prettier-2')};
+        module.exports = {prettierPath: require.resolve('prettier')};
```

Comment:
This test should running against the `prettier/index.cjs`, maybe better run against `prettier/index.mjs` too.

Reply:
Since `require` will be called on the main thread anyway, I will use `index.cjs`.